scripts/test.sh

```diff
@@ -20,6 +20,12 @@ set -e
 echo "\n\nBuild projects..."
 pnpm run build
 
+# Test cargo projects
+echo "\n\nCargo test..."
+cargo test
+echo "\n\nForc test..."
+forc test
+
```

Comment:
We don''t have any in-language tests. Keep or remove?

Reply:
doesn't really add much overhead to leave it here for now, and makes it easier to use in-language tests in the future